libraries/bot-integration-core/pom.xml

```diff
@@ -65,6 +65,11 @@
       <groupId>com.microsoft.bot</groupId>
       <artifactId>bot-builder</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
```

Comment:
This is needed or was an auto-addition of the IDE? Don't remember needing this for the previous tests classes

Reply:
This is necessary, if we don't add this dependecy, we cannot use Mockito in the unit tests of this library